assets/components/productPage/productPagePeriodForm/productPagePeriodFormLabel.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React, { type Node } from 'react';
+import SvgCheckmark from 'components/svgs/checkmark';
+
+import uuidv4 from 'uuid';
+
+
+// ---- Types ----- //
+
+type PropTypes = {|
+  type: string,
+  title: string,
+  offer: string | null,
+  children: Node,
+  checked: boolean,
+  onChange: (Event) => void,
+|};
+
+
+// ----- Render ----- //
+
+export default ({
+  type, title, offer, children, checked, onChange,
+}: PropTypes) => {
+  const id = uuidv4();
+  return (
+    <label onChange={onChange} htmlFor={id} className="component-product-page-period-form-label-wrap">
```

Comment:
The top-level `className` takes the name of the component, in this case `component-product-page-period-form-label`.